docs/tutorial/using-checkbox/checkbox-commands.rst

```diff
@@ -0,0 +1,296 @@
+    you have to explicitly use it.
+
+List the content of a test plan as executed by Checkbox
+=======================================================
+
+In the previous section, you've listed all the test
+plans related to audio. Select one of them, for instance
+``com.canonical.certification::audio-cert-automated``, and see what jobs
+it contains:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli list-bootstrapped com.canonical.certification::audio-cert-automated
+
+    com.canonical.plainbox::manifest
+    com.canonical.certification::package
+    com.canonical.certification::audio/detect_sinks
+    com.canonical.certification::device
+    com.canonical.certification::audio/detect_sources
+    com.canonical.certification::audio/alsa_record_playback_automated
+    com.canonical.certification::audio/alsa_info_collect
+    com.canonical.certification::audio/alsa_info_attachment
+    com.canonical.certification::audio/list_devices
+    com.canonical.certification::audio/valid-sof-firmware-sig
+
+If you were to run this test plan with Checkbox, it would run these jobs in
+the order shown above.
+
+.. note::
+
+    The name of this command refers to the Checkbox :term:`bootstrapping`
+    phase.
+
+But what are these jobs exactly? You can use the ``show`` command to see the
+content of a Checkbox object.
+
+Show the content of a Checkbox object
+=====================================
+
+Have a look at the ``com.canonical.certification::audio/list_devices`` job
+listed in the ``com.canonical.certification::audio-cert-automated`` test
+plan above, for instance. What is it exactly? What does it contain? What is
+its **definition?** Use the ``show`` command to find out:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli show com.canonical.certification::audio/list_devices
+
+    origin: /snap/checkbox22/current/providers/checkbox-provider-base/units/audio/jobs.pxu:1-9
+    plugin: shell
+    category_id: com.canonical.plainbox::audio
+    id: audio/list_devices
+    estimated_duration: 1.0
+    requires:
+     device.category == 'AUDIO'
+     package.name == 'alsa-base'
+    command: cat /proc/asound/cards
+    _description: Test to detect audio devices
+
+The first line tells you that this job comes from lines 1 to 9 of the
+``/snap/checkbox22/current/providers/checkbox-provider-base/units/audio/jobs.pxu``
+file. The other lines show its definition. We can see what it does ("Test to
+detect audio devices"), how (using the ``cat /proc/asound/cards`` command),
+and many other details.
+
+.. note::
+
+    Each of the fields shown in a job definition are explained in the
+    :ref:`job` page.
+
+Now that you know the definition of this job, you can run it to see what
+the output generated by Checkbox look like.
+
+Run a particular test plan or a set of jobs
+===========================================
+
+Run the following command:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli run .*audio/list_devices
```

Comment:
Minor: surround regex with `"`

Reply:
I hesitated to do that, because it works as is in Bash, but it does not in Fish. I'll add `"`.

Reply:
Fixed in [20d9258](https://github.com/canonical/checkbox/pull/635/commits/20d92587d4c1323c44666a647f24b991ccabe62d)